lib/classes/PluginManager.js

```diff
@@ -23,7 +23,14 @@ const requireServicePlugin = (servicePath, pluginPath, localPluginsPath) => {
       if (!isModuleNotFoundError(error, absoluteLocalPluginPath)) throw error;
     }
   }
-  return require(cjsResolve(servicePath, pluginPath).realPath);
+  try {
+    return require(cjsResolve(servicePath, pluginPath).realPath);
+  } catch (error) {
+    if (!isModuleNotFoundError(error, pluginPath)) throw error;
```

Comment:
Sorry just noticed, We should add following to condition ` || pluginPath.startsWith('.')`.

As global `node_modules` search should occur only for _package_ requires (not relative modules)